modules/gitops-release-notes-1-10-1.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-10-1_{context}"]
+= Release notes for {gitops-title} 1.10.1
+
+{gitops-title} 1.10.1 is now available on {OCP} 4.12 and 4.13.
+
+[id="errata-updates-1-10.1_{context}"]
+== Errata updates
+
+[id="gitops-1-10-1-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX - {gitops-title} 1.10.1 security update advisory
+
+Issued: 2023-10-30
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://errata.engineering.redhat.com/advisory/122170[RHSA-2023:122170]
+//* link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]
```

Comment:
I'm guessing this is just placeholder until the real live-ID and URL is available, but I might caution from having the internal URL here in the PR, even if only temporary.

Reply:
Noted, thank you.